### ISSUE TITLE
Feature Request: add validation loss to tf.summary outputs

### ISSUE THREAD

**Issue by carlogrisetti** [uber/ludwig#350]
**Is your feature request related to a problem? Please describe.**
I am not able to evaluate a model validation loss history and\or track the model validation loss evolution when not having access to the CLI and have to rely to the train_reg_mean_loss graph to deduce what the validation loss could be trending to. I often find myself with a continuously improving training loss, but a stuck validation loss (hence, the model is not getting better and not being saved).
I can only find out about this when looking at the CLI and finding out that there has not been an improvement in the last X epochs. Being able to see the validation loss graph not having improved in the last X epochs could lead me to stop the training and try a different approach, without having to in any case remote into the training machine, but allowing for a web-based "diagnostic".

**Describe the use case**
Being able to track validation loss from the existing TensorBoard interface, since when having small datasets and not having the capability to augment input data, the validation set (thus the validation loss) could become a very separate thing from the training set (and its loss value).

**Describe the solution you'd like**
Adding the validation loss measure already calculated in the CLI as a tf.summary output, alongside the existing train_reg_mean_loss

**Comment by w4nderlust** [comment]
While training Ludwig keeps track of loss and measures and at the end it saves them in `training_statistics.json` so you can find there what you are looking for. Moreover there is also a  visualization called ` learning_curves` that show you losses and measures on training and validation over the course of the training.
So I believe this use case is already covered.

**Comment by carlogrisetti** [comment]
Think about a long running training. I should remote into the machine, stop the training, run the visualization scripts and then decide to resume or not the training.
My intent is to monitor the training whilist doing other things (like in meetings or away from work), and I can only write some sort of parser for the training_statistics.json file, but having tf.summary available (and already used for the training statistics) seems really to be a waste...
I would like to open a remote session only to actually work on that (even just for stopping the training) but not each time I want to see how that's going.

If this doesn't cut it, ok for me, I just wanted to better explain the use case

**Comment by w4nderlust** [comment]
If you are keeping track of progress, you can you the training_rpogress.json file in the model dir that gets updated every epoch. For reading the validation loss you just read the json and then `loaded_json['vali_stats'][your_output_feature_name]['loss']`.

**Comment by carlogrisetti** [comment]
If and when I'm able to sort out how to eventually implement this, would you agree in having that value as an additional output in the existing logfiles alongside the train_reg_mean_loss ?

**Comment by w4nderlust** [comment]
The way to do it is the following: https://stackoverflow.com/questions/34471563/logging-training-and-validation-loss-in-tensorboard
But, as in Ludwig the evaluation is performed with the same subgraph on training validation and test set at the end of the epoch, you would have to figure out a clean way to do it.
Anyway, I believe there's no actual need as the values you want are already available to you through other means.
